nameko/testing/services.py

```diff
@@ -476,8 +476,15 @@ class Once(Entrypoint):
     the service container started.
     """
     def __init__(self, *args, **kwargs):
+        expected_exceptions = kwargs.pop('expected_exceptions', ())
```

Comment:
The `once` entrypoint is a bit special in that is passes `**kwargs` directly to the entrypoint. If you don't pop them off you get unexpected argument errors.